seesaw/script/run_pipeline.py

```diff
@@ -164,7 +164,12 @@ def check_and_update():
 
 
 def main():
-    parser = ArgumentParser(description="Run the pipeline")
+    global runner
```

Comment:
Is this required? I don't see an outer reference to `runner` in the changes, and `main` initializes its own runner later on.